main.pm

```diff
@@ -719,6 +719,9 @@ sub load_x11tests() {
         && !system_is_livesystem)
     {
         loadtest "x11/gnucash.pm";
+        loadtest "x11/xchat.pm";
+        loadtest "x11/hexchat.pm";
+        loadtest "x11/vlc.pm";
```

Comment:
just check the media I owned here, not the last snapshot release, vlc does not in the media, hexchat and xchat does.